### ISSUE TITLE
Sprint 1.31.0 2/2 - 29 Nov - 10 Dec

### ISSUE THREAD

**Issue by timgl** [PostHog/posthog#7338]
# Global Sprint Planning

## Retro: Status of Outcomes from Previous Sprint

1. Deploy groups **In progress, still a lot of stuff to do. Surprising that Neil was support hero, so lost person weeks.**
2. Increased number of discoveries of retentions. **Still working on a couple of things**
3. e2e helm chart testing set up for GCP/AWS/DO **Completed GCP/DO, we will complete AWS, blocked on Azure. Able to spot quite a few bugs thanks to tests**
4. MVP big migration support **Pull request will be done for mvp. Support hero and customer support**
5. Fix all the known person data integrity issues **Tiina was secondary for both weeks, plus customer upgrade**

6. Collaboration: Sharable urls (with team_id) @Twixes  **De-prioritised for this sprint in favour of design work/navigation**
7. Perfect discovery and playback experience for session recordings @rcmarron  **Some progress, some issues on the session recording list. Still work to do.**

## Retro: What can we do better next sprint?

1. Support hero confusion
2. Clearer/more specific goals (collaboration)
3. Big goals focussed on problems rather than solutions
4. Lots of support

## Plan: Proposed Goals for Next Sprint 

Each goal should have a single owner. Owner can only be an engineer.

Core Analytics
1. Actor modal
2. Defining experiments with a clear goal (e.g. conversion, retention, volume, ...) and three clear outcomes (success, failure, inconclusive) that can be quickly identified.

Core Experience
3. "make insights and dashboards not a frustrating experience anymore" @paolodamico 
4. "perfect playback experience for session recordings" @rcmarron 

Platform
5. Release system to do events table migration
6. Upgrade Helm chart to k8s 1.22

# Team sprint planning

For your team sprint planning copy this template into a comment below for each team.

```
Team ___

## Retro

<!-- Talk about what went well, what didn't go well and any actions to improve next time -->

- 

## Hang over items from previous sprint

<!-- For each item, decide to re-prioritise (and add below) or deprioritise -->

- Item 1. prioritised/deprioritise

## Planning

<!-- Each item should have a single owner. Owner can only be an engineer. -->

### High priority

-

### Low priority / side quests

-

```

**Comment by mariusandra** [comment]
## Team Core Experience planning

- Rick: "the session recordings experience is still suboptimal" (window_id, ~~linking insights to recordings~~)
- Paolo: "insights and dashboards are a frustrating experience" https://github.com/PostHog/posthog/issues/6513
- Michael: "not being able to share links to anything" https://github.com/PostHog/posthog/issues/7245
- Alex: ~~"posthog can't notify people of new stuff" (needed for insight discussions)~~ join Paolo and Michael
- Paul: ~~"breakdown isn't finished"~~ join Paolo and Michael
- Chris: do all the things
![image](https://user-images.githubusercontent.com/53387/143275850-6edd7ff7-698e-4c87-8faa-35236af40b42.png)

**Comment by yakkomajuri** [comment]
# Team Platform

## Retro

- @guidoiaquinti: most impactful sprint so far, consequence of well-scoped goal and successful planning
- @tiina303: spent a lot of time on deployments support, would like to have more time to work on code (data integrity)
- @yakkomajuri: used support hero week to ship improvements to plugins, a lot of postgres -> ch migration support, good progress on big database migrations

## Hang over items from previous sprint

- **Fix all the known person data integrity issues**: re-prioritise (blocked on #6667, last mile on #6834)

## Planning

@fuziontech and @tiina303 are support heroes.

### High priority

- Release system to do events table migration (@yakkomajuri)
- Upgrade Helm chart to k8s 1.22 (@guidoiaquinti)
- Last mile on #6834 (@tiina303)

### Low priority / side quests

- #6662 
- https://github.com/orgs/PostHog/projects/10

**Comment by EDsCODE** [comment]
## Team Core Analytics

### Retro (main items)

- @EDsCODE actor modal (consolidating groups and persons) for trends and funnels
- @macobo groups feature flagging
- @hazzadous adjustments to retention experience. Drafts of retention breakdown j curves
- @neilkakkar support hero
- @liyiy out of office

### Planning

Experimentation MVP — @neilkakkar @liyiy 

Groups wrap up — @macobo 

- Groups pages
- Queries (recordings)
- Docs! 
- Upsell & design changes
- Blog post/launch support
- Library support (ruby, -go feature flags support)


Actor modal for rest of insights — @EDsCODE 

- Add support for paths, retention, + all other insights
- Save as cohort functionality for all insights
- Stretch goal: save as CSV


Retention enhancement (estimating 1 week) — @hazzadous 

- J curve support
- Breakdowns

**Comment by paolodamico** [comment]
Alright, sprint done!
